scripts/infra-mcp/tools/get_container_tags.py

```
@@ -0,0 +1,501 @@
+                    tag_data.append({
+                        'name': tag['name'],
+                        'last_updated': tag.get('last_updated'),
+                        'size': tag.get('full_size', 0),
+                        'digest': arch_digest
+                    })
+
+            # Sort by last_updated in descending order (newest first)
+            def _iso(dt_str):
+                try:
+                    return datetime.fromisoformat(dt_str.replace('Z', '+00:00'))
+                except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)